<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiResultWindow</class>
 <widget class="QMainWindow" name="MultiResultWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="resultpic">
   <property name="styleSheet">
    <string notr="true">#resultpic{background-image: url(:/images/resultwindow/result_background.png);}</string>
   </property>
   <widget class="QPushButton" name="btnReturn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>610</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnReturn{border-image:url(:/images/resultwindow/back.png)}
QPushButton::pressed#btnReturn{border-image:url(:/images/resultwindow/back-.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>-100</y>
      <width>600</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:100px;
	color:rgb(70, 18, 53);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>Time's up!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnUpdate{border-image:url(:/images/resultwindow/update.png)}
QPushButton::pressed#btnUpdate{border-image:url(:/images/resultwindow/update-.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cnt1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:rgb(178, 96, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="score">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>130</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:rgb(4, 0, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="board_1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cnt5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>530</y>
      <width>40</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:15px;
	color:rgb(178, 96, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="board_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>520</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="board">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>270</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/score.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="board_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>250</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cnt2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:rgb(178, 96, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="board_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>350</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cnt3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:rgb(178, 96, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="board_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>450</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="cnt4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:rgb(178, 96, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="username_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px;
color:rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_board_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="username_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px;
color:rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_board_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="username_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px;
color:rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_board_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="username_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:30px;
color:rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_board_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>220</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="my_cnt_1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>230</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:rgb(178, 96, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="board_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>240</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="board_7">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>240</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="my_cnt_2">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>230</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:rgb(178, 96, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="board_8">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>380</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="my_cnt_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>370</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:rgb(178, 96, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="board_9">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>380</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/singlewindow/skill-.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="my_cnt_4">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>370</y>
      <width>90</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size:30px;
	color:rgb(178, 96, 255);
	font-family:幼圆;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="shadow_1">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/gamewindow/shadow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="shadow_2">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>240</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/gamewindow/shadow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="shadow_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>380</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/gamewindow/shadow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="shadow_4">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>380</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resource/res.qrc">:/images/gamewindow/shadow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>name_board_4</zorder>
   <zorder>name_board_3</zorder>
   <zorder>name_board_2</zorder>
   <zorder>board_5</zorder>
   <zorder>cnt5</zorder>
   <zorder>title</zorder>
   <zorder>board_1</zorder>
   <zorder>btnReturn</zorder>
   <zorder>btnUpdate</zorder>
   <zorder>cnt1</zorder>
   <zorder>board</zorder>
   <zorder>score</zorder>
   <zorder>board_2</zorder>
   <zorder>cnt2</zorder>
   <zorder>board_3</zorder>
   <zorder>cnt3</zorder>
   <zorder>board_4</zorder>
   <zorder>cnt4</zorder>
   <zorder>name_board_1</zorder>
   <zorder>username_1</zorder>
   <zorder>username_2</zorder>
   <zorder>username_3</zorder>
   <zorder>username_4</zorder>
   <zorder>board_6</zorder>
   <zorder>my_cnt_1</zorder>
   <zorder>board_7</zorder>
   <zorder>my_cnt_2</zorder>
   <zorder>board_8</zorder>
   <zorder>my_cnt_3</zorder>
   <zorder>board_9</zorder>
   <zorder>my_cnt_4</zorder>
   <zorder>shadow_1</zorder>
   <zorder>shadow_2</zorder>
   <zorder>shadow_3</zorder>
   <zorder>shadow_4</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../../resource/res.qrc"/>
 </resources>
 <connections/>
</ui>
